mongo/mongo.go

```diff
@@ -238,13 +238,20 @@ var (
 		Patch:         "",
 		StorageEngine: WiredTiger,
 	}
-	// Mongo406wt represents 'mongodb-server-core' at version 4.0.6 with WiredTiger
+	// Mongo406wt represents snap 'juju-db' at version 4.0.x with WiredTiger
+	// XXX: (jam) This is the only Mongo declaration that declares a Point
+	//  release. It doesn't seem the right place to do so. We currently have to,
+	//  because the K8s pod spec requires it but
+	//  a) It doesn't match the current snap version
+	//  b) It prevents our normal "update the package to get an updated mongo without
```

Comment:
That's by design so we get fully repeatable deploys.
But maybe that's too restrictive. Maybe we just want to tag upstream mongo images as juju-db:4.X and feel free to copy a point release across when needed.